src/applications/vaos/services/appointment/transformers.v2.js

```diff
@@ -36,16 +39,75 @@ function getTypeOfCareById(id) {
 /**
  * Gets the type of visit that matches our array of visit constant
  *
- * @param {Object} id VAR appointment object
+ * @param {Object} id VAOS Service appointment object
  * @returns {String} type of visit string
  */
 function getTypeOfVisit(id) {
   return TYPE_OF_VISIT.find(type => type.id === id)?.name;
 }
 
+/**
+ * Finds the datetime of the appointment depending on the appointment type
+ * and returns it as a moment object
+ *
+ * @param {Object} appt VAOS Service appointment object
+ * @returns {Object} Returns appointment datetime as moment object
+ */
+function getMomentConfirmedDate(appt) {
+  if (appt.kind === 'cc' && appt.timeZone) {
+    const zoneSplit = appt.timeZone.split(' ');
+    const offset = zoneSplit.length > 1 ? zoneSplit[0] : '+0:00';
+    return moment
+      .utc(appt.appointmentTime, 'MM/DD/YYYY HH:mm:ss')
+      .utcOffset(offset);
+  }
+
+  const timezone = getTimezoneBySystemId(appt.locationId)?.timezone;
+
+  return timezone ? moment(appt.start).tz(timezone) : moment(appt.start);
+}
+
+/**
+ *  Determines whether current time is less than appointment time
+ *  +60 min or +240 min in the case of video
+ * @param {*} appt VAOS Service appointment object
+ */
+export function isPastAppointment(appt) {
+  const isVideo = appt.kind === 'telehealth';
+  const threshold = isVideo ? 240 : 60;
+  const apptDateTime = moment(getMomentConfirmedDate(appt));
+  return apptDateTime.add(threshold, 'minutes').isBefore(moment());
+}
+
+/**
+ * Gets the atlas location and sitecode
+ *
+ * @param {Object} appt VAOS Service appointment object
+ * @returns {String} returns format data
+ */
+function getAtlasLocation(appt) {
+  const atlas = appt.telehealth.atlas;
+  return {
+    id: atlas.siteCode,
+    address: {
+      line: [atlas.address.streetAddress],
+      city: atlas.address.city,
+      state: atlas.address.state,
+      postalCode: atlas.address.zipCode,
+    },
+    position: {
+      longitude: atlas.address.longitude,
+      latitude: atlas.address.latitude,
+    },
+  };
+}
+
 export function transformVAOSAppointment(appt) {
   const isCC = appt.kind === 'cc';
   const isVideo = appt.kind === 'telehealth';
+  const isAtlas = appt.telehealth?.vvsKind === 'ADHOC';
```

Comment:
I think this check should be something like `!!appt.telehealth.atlas`. There are video appointments that are `ADHOC`, but not ATLAS.